### ISSUE TITLE
Before publishing Linux GUI client

### ISSUE THREAD

**Issue by ReactorScram** [CloudFire-LLC/cloudfire#3884]
From #3694

```[tasklist]
- [x] Test Linux + Windows artifacts from CI+CD
- [x] Paths for config, logs, device ID, etc.
- [x] Fix artifact names in update check (For Windows, too) (https://github.com/firezone/firezone/pull/3769)
- [x] Fix crash handling (https://github.com/firezone/firezone/pull/3737)
- [x] Ubuntu 22.04.4 in CI just doesn't like Tauri (https://github.com/firezone/firezone/pull/3770)
- [x] Fix / test token storage (#4799)
- [x] Keyring doesn't like the CI, and some users may not have a secret store installed
- [x] Deep links
- [x] Ensure IPC socket is in a secure path (See inetd advice under "Unix Domain Sockets" https://man.freebsd.org/cgi/man.cgi?inetd(8))
- [x] Ensure "Cancel sign-in" works while waiting on browser
- [x] Notifications (#4820)
- [x] Permissions / startup
- [x] Make sure GUI and IPC service logs are both going somewhere useful (#4823)
- [x] (postponed) ~~Ensure single-instance still works #4790~~
- [x] (postponed) ~~Show useful error if we try to sign in but the IPC service isn't up~~
- [ ] #4083
- [x] (postponed) ~~Address technical debt (#3636)~~
- [ ] #4875
- [x] Install script #4879
- [ ] https://github.com/firezone/firezone/pull/4881
- [x] Update manual testing checklist #4882
- [ ] #4822
- [x] Make sure connlib logs end up in exported zips #4830
- [ ] #4876
```

**Comment by ReactorScram** [comment]
https://github.com/firezone/firezone/releases/tag/1.0.3
